test/metrics/metrics_test.cpp

```diff
@@ -199,6 +213,56 @@ TEST_F(MetricsTests, TransactionCSVTest) {
   metrics_manager_->UnregisterThread();
 }
 
+/**
+ *  Testing logging query trace metrics, single thread
+ */
+// NOLINTNEXTLINE
+TEST_F(MetricsTests, QueryCSVTest) {
+  for (const auto &file : metrics::QueryTraceMetricRawData::FILES) unlink(std::string(file).c_str());
+  const settings::setter_callback_fn setter_callback = MetricsTests::EmptySetterCallback;
+  auto action_context = std::make_unique<common::ActionContext>(common::action_id_t(1));
+  settings_manager_->SetBool(settings::Param::metrics_query_trace, true, common::ManagedPointer(action_context),
+                             setter_callback);
+
+  db_main_->GetNetworkLayer()->GetServer()->RunServer();
+
+  try {
+    pqxx::connection connection(fmt::format("host=127.0.0.1 port={0} user={1} sslmode=disable application_name=psql",
+                                            port_, catalog::DEFAULT_DATABASE));
+
+    pqxx::work txn1(connection);
+    txn1.exec("CREATE TABLE TableA (id INT PRIMARY KEY, data TEXT);");
+    txn1.exec("INSERT INTO TableA VALUES (1, 'abc');");
+    txn1.exec("SELECT * FROM TableA");
+    txn1.commit();
+  } catch (const std::exception &e) {
+    EXPECT_TRUE(false);
+  }
+
+  std::this_thread::sleep_for(std::chrono::seconds(1));
+
+  metrics_manager_->Aggregate();
+  const auto aggregated_data = reinterpret_cast<QueryTraceMetricRawData *>(
+      metrics_manager_->AggregatedMetrics().at(static_cast<uint8_t>(MetricsComponent::QUERY_TRACE)).get());
+  EXPECT_NE(aggregated_data, nullptr);
+  EXPECT_GE(aggregated_data->query_trace_.size(), 2);  // 2 data point recorded
```

Comment:
Should this be `EXPECT_GE` or `EXPECT_EQ`?

Reply:
Fixed it to be `EXPECT_EQ`.